apps/api/src/app/change/usecases/change-enabled/change-enabled.usecase.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { ChangeEntityTypeEnum, ChangeRepository } from '@novu/dal';
+import { diffApply } from '../../utils';
+import { ChangeEnabledCommand } from './change-enabled.command';
+import { EnvironmentRepository } from '../../../../../../../libs/dal/src/repositories/environment/environment.repository';
```

Comment:
We have the ../libs/dal improt here aswell :) Can we add a rule to .eslintrc.js to ban imports from "../../libs/dal/src",  "../../libs/shared/src"?